tests/framework/clients/rancher/v1/client.go

```diff
@@ -219,11 +219,11 @@ func (c *SteveClient) Delete(container *SteveAPIObject) error {
 // SteveAPIObject to its native kubernetes type
 // e.g. converting a SteveAPIObject spec to a NamespaceSpec
 //
-//	 namespaceSpec := &coreV1.NamespaceSpec{}
-//   spec, err := namespaces.ConvertSpecOrStatusType(createdNamespace.Spec, namespaceSpec)
-//   require.NoError(p.T(), err)
+//	namespaceSpec := &coreV1.NamespaceSpec{}
+//	spec, err := namespaces.ConvertToK8sType(createdNamespace.Spec, namespaceSpec)
+//	require.NoError(p.T(), err)
 //
-//   spec.(*coreV1.NamespaceSpec).Finalizers
+//	namespaceSpec.Finalizers
```

Comment:
Did you fix the comments Ugur? Or was that not merged correctly?

Reply:
Yes  I did fix that, the explanation was for the previous iteration

Reply:
Ok cool, thank you. Rebase your branch as well since the provisioning integration tests have been fixed. And with the freeform runs I'll approve it

Reply:
Found one line that I missed updating for this, L223, and updated that one as well